mmengine/infer/infer.py

```diff
@@ -380,11 +380,48 @@ def _load_model_from_metafile(self, model: str) -> Tuple[Config, str]:
                 model_aliases = [alias.lower() for alias in model_aliases]
             if (model_name == model or model in model_aliases):
                 cfg = Config.fromfile(
-                    osp.join(package_path, '.mim', model_cfg['Config']))
+                    osp.join(config_dir, model_cfg['Config']))
                 weights = model_cfg['Weights']
                 weights = weights[0] if isinstance(weights, list) else weights
                 return cfg, weights
-        raise ValueError(f'Cannot find model: {model} in {project}')
+        raise ValueError(f'Cannot find model: {model} in {self.scope}')
+
+    @staticmethod
+    def _get_config_dir(scope):
+        """Get the directory of the ``Config`` when package is installed or
+        ``PYTHONPATH`` is set.
+
+        Args:
+            scope (str): The scope of repository.
+
+        Returns:
+            str: The directory of the ``Config``.
+        """
+        project = MODULE2PACKAGE[scope]
+        try:
+            module = importlib.import_module(scope)
+        except ImportError as e:
+            if scope not in MODULE2PACKAGE:
```

Comment:
If the `scope` does not in `MODULE2PACKAGE`, the clause `project = MODULE2PACKAGE[scope]` will throw an error first.